tests/test_pipeline.py

```diff
@@ -43,6 +47,32 @@ def test_delete_pipeline(client):
         assert rep.status_code == 200
 
 
+def test_create_pipeline(client, default_scenario):
+    pass
+    # pipelines_url = url_for("api.pipelines")
+    # rep = client.post(pipelines_url, json={})
+    # assert rep.status_code == 400
+    # assert rep.json == {"message": "Scenario id is missing."}
+
+    # pipelines_url = url_for("api.pipelines")
+    # rep = client.post(pipelines_url, json={"scenario_id": "SCENARIO_scenario_id"})
+    # assert rep.status_code == 400
+    # assert rep.json == {"message": "Pipeline name is missing."}
+
+    # pipelines_url = url_for("api.pipelines")
+    # rep = client.post(pipelines_url, json={"scenario_id": "SCENARIO_scenario_id", "pipeline_name": "pipeline"})
+    # assert rep.status_code == 404
+
+    # _ScenarioManagerFactory._build_manager()._set(default_scenario)
+    # with mock.patch("taipy.core.scenario._scenario_manager._ScenarioManager._get") as config_mock:
+    #     config_mock.return_value = default_scenario
+    #     pipelines_url = url_for("api.pipelines")
+    #     rep = client.post(
+    #         pipelines_url, json={"scenario_id": default_scenario.id, "pipeline_name": "pipeline", "tasks": []}
+    #     )
+    #     assert rep.status_code == 201
```

Comment:
This must be uncommented and updated

Reply:
oh shoot, I forgot to uncomment the tests!!